src/gromo/containers/growing_dag.py

```diff
@@ -432,8 +433,8 @@ def calculate_bottleneck(
 
         Parameters
         ----------
-        generations : list[dict]
-            list of dictionaries with growth actions information
+        generations : list[Expansion]
```

Comment:
Could we use a better name than generation? Like growth_history?

Reply:
It's more a list of possible actions but I can find a better name for it.